src/wasm-interpreter.h

```diff
@@ -170,11 +210,36 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
   }
 
 public:
-  ExpressionRunner(Index maxDepth) : maxDepth(maxDepth) {}
+  ExpressionRunner(Flags flags = FlagValues::DEFAULT,
+                   Index maxDepth = 0,
+                   Index maxLoopIterations = 0)
```

Comment:
how about removing the `= 0`s here, so that we have (as suggested earlier) just a single place with `= 0`, earlier. For here we don't need a default value. But let's add a plain `ExpressionRunner() {}` constructor, which then would use those defaults.

Reply:
(or maybe a constructor that only takes the flags? depends on what we need. preferably as few constructors as possible, but also without those `= 0`s duplicated)

Reply:
Made this three ctors now:

1. A plain one because it's good to have
2. One taking just `maxDepth`, which is what `ConstantExpressionRunner` and `RuntimeExpressionRunner` use
3. One taking all of `module`, `flags`, `maxDepth` and `maxLoopIterations` which is what `Precompute` and the C-API use

plus making the `0` a `NO_LIMIT` constant for clarity in code. That ok? :)